pkg/api/reporting/types/types.go

```diff
@@ -11,6 +11,12 @@ type ReportingInfo struct {
 	K8sVersion         string         `json:"k8s_version"`
 	KOTSInstallID      string         `json:"kots_install_id"`
 	KURLInstallID      string         `json:"kurl_install_id"`
+	GitOpsReport       *GitOpsReport  `json:"gitops_report"`
+}
+
+type GitOpsReport struct {
```

Comment:
can we not create a new type for this and just have it be part of the original `ReportingInfo` type?